app/time_tracking/work_statistic/tests.py

```diff
@@ -0,0 +1,95 @@
+from django.contrib.auth import get_user_model
```

Comment:
test cases in here are not enough, and not designed to test valuable cases (no unit test for `work_on_leav_time`)

Reply:
Strongly agree.